### PR TITLE
updated recommend function to blacklist classes with same title as input and input classes

### PR DESCRIPTION
**Pull request by TiloRC**

### CODE REVIEW

**Comment by saatvikkher** [comment]
Add screenshots of the site being tested locally.

**Review by saatvikkher — CHANGES_REQUESTED** [review]
Do not commit metadata. Make sure you only commit necessary files. There's 10 changed files in the commit and many are unnecessary/metadata.

File: app.py [hunk 1]
```diff
@@ -48,7 +52,11 @@ def index():
 @app.route('/rec',methods=['POST'])
 def getvalue():
     coursename = request.form['search'].split(" ")[0]
-    df = recommend(coursename)
+
+    #print(recommend2([coursename]))
```
Comment (saatvikkher) [inline app.py:56]: No commented-out print statements in main.

File: recommend_GloVe/.#recommend_GloVe.py [hunk 2]
```diff
@@ -0,0 +1 @@
+reneau-cardoso@times-mbp.local.87556
```
Comment (saatvikkher) [inline recommend_GloVe/.#recommend_GloVe.py]: This file shouldn't be here.

**Comment by TiloRC** [comment]
Removed meta data files and checked to see if the flask app still ran after changes

**Comment by TiloRC** [comment]
Noticed some other bugs. Will work on this later.

**Pull request closed without merging**